### PR TITLE
chore: update documentation

### PR DESCRIPTION
**Pull request by jschfflr**
Explain the relationship between `page.setOfflineMode` and `page.emulateNetworkConditions`.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -1704,7 +1704,8 @@ await page.evaluate(() => matchMedia('print').matches);
   - `latency` <[number]> Latency (ms), `0` to disable
 - returns: <[Promise]>
 
-> **NOTE** This does not affect WebSockets and WebRTC PeerConnections (see https://crbug.com/563644)
+> **NOTE** This does not affect WebSockets and WebRTC PeerConnections (see https://crbug.com/563644). To set the page offline, you can use [page.setOfflineMode(enabled)](#pagesetofflinemodeenabled).
```
Comment (OrKoN) [inline docs/api.md:1709]: This part is not auto-generated right?

Comment (jschfflr) [reply docs/api.md:1709]: No, I added it manually. Is `api.md` now generated from code?

Comment (OrKoN) [reply docs/api.md:1709]: I am not sure, I thought so but I might be wrong :D

**Review by OrKoN — APPROVED** [review]

File: docs/api.md (hunk 1 above)
Comment (jschfflr) [reply docs/api.md:1709]: The same strings can be found in `Page.ts` - I updated them too to keep them in sync.
@mathiasbynens Do we already generate documentation from source automatically?

**Pull request merged**